.circleci/config.yml

```diff
@@ -27,7 +27,7 @@ jobs:
           git checkout $CIRCLE_BRANCH || git checkout master
           export WEBAPP_GIT_COMMIT=$(git rev-parse HEAD)
           echo "$WEBAPP_GIT_COMMIT"
-          curl -f -o ./dist.tar.gz https://releases.mattermost.com/mattermost-webapp/commit/${WEBAPP_GIT_COMMIT}/mattermost-webapp.tar.gz && mkdir ./dist && tar -xvf ./dist.tar.gz -C ./dist --strip-components=1 || make node_modules test build
+          npm install && cd node_modules/mattermost-redux && npm install && npm run build && cd ../.. && make build
```

Comment:
@cpanato Do we need the curl option? This is the one causing the build to break because redux does not exist.

Reply:
we dont need, you can refactor as you need

Reply:
you can use wget if is better